commands/provider/service_deployer.go

```diff
@@ -115,7 +115,7 @@ func (p *ServiceProvider) deployServiceFromMarketplace(u string, env map[string]
 	case "https", "http":
 		url = source
 	case "ipfs":
-		url = "https://gateway.ipfs.io/ipfs/" + source
+		url = "http://ipfs.app.mesg.com/ipfs/" + source
```

Comment:
This is not the right address...
```suggestion
		url = "http://ipfs.app.mesg.com:8080/ipfs/" + source
```